crates/schema/src/def/validate/v9.rs

```diff
@@ -933,7 +1027,24 @@ fn check_scheduled_reducers_exist(
         .collect_all_errors()
 }
 
-fn proccess_misc_exports(
+fn check_function_names_are_unique(
+    reducers: &IndexMap<Identifier, ReducerDef>,
+    procedures: &IndexMap<Identifier, ProcedureDef>,
+) -> Result<()> {
+    let names = reducers.keys().collect::<HashSet<_>>();
```

Comment:
Btw, do we check anywhere that the set of reducer names are internally unique? Same for tables? If not, this is probably a good place to do it for both reducers and procedures, and then you can use `.intersection(_)`